test/test_fakeredis.py

```diff
@@ -1,30 +1,33 @@
-from time import sleep, time
-
-import pytest_asyncio
-from redis.exceptions import ResponseError
-from collections import OrderedDict
-import os
+import logging
 import math
+import os
 import threading
-import logging
+from collections import OrderedDict
+from datetime import datetime, timedelta
 from queue import Queue
+from time import sleep, time
 
-import six
-from packaging.version import Version
 import pytest
+import pytest_asyncio
 import redis
 import redis.client
+import six
+from packaging.version import Version
+from redis.exceptions import ResponseError
 
 import fakeredis
-from datetime import datetime, timedelta
-
 
 REDIS_VERSION = Version(redis.__version__)
-REDIS3 = REDIS_VERSION >= Version('3')
-
 
-redis2_only = pytest.mark.skipif(REDIS3, reason="Test is only applicable to redis-py 2.x")
-redis3_only = pytest.mark.skipif(not REDIS3, reason="Test is only applicable to redis-py 3.x")
+redis_below_v3 = pytest.mark.skipif(REDIS_VERSION >= Version('3'), reason="Test is only applicable to redis-py 2.x")
```

Comment:
`REDIS_VERSION >= Version('3')` check occurs 4 times in this file; maybe to leave this as a local variable instead, e.g. `REDIS3PLUS = REDIS_VERSION >= Version('3')`?